scripts/test.sh

```diff
@@ -19,10 +19,9 @@ service mysql start
 
 export SQLFLOW_TEST_DB=mysql
 
+source activate sqlflow-dev
```

Comment:
Maybe don't need souce pyenv here? It's already configurated in the Dockerfile: https://github.com/sql-machine-learning/sqlflow/pull/494/files#diff-3254677a7917c6c01f55212f86c57fbfR11

Reply:
Remove this will cause CI error. Not sure why.

Reply:
@typhoonzero https://github.com/sql-machine-learning/sqlflow/pull/494/files#r291730654